### ISSUE TITLE
Feature Request - fullscrean switch to another stream

### ISSUE THREAD

**Issue by filippcanadian** [AlexxIT/WebRTC#702]
Hi AlexxIT and Community!
Is there any way to turn on another (in my case it is main stream of camera) stream clicking fullscrean button?

**Comment by AlexxIT** [comment]
No. But you can use `streams` config for switching different streams in one card.
